fn_ip_void/README.md

```diff
@@ -0,0 +1,125 @@
+# IPVOID Search Function for IBM Resilient
+
+## Table of Contents
+  - [app.config settings](#appconfig-settings)
+  - [Function Inputs](#function-inputs)
+  - [Function Output](#function-output)
+  - [Pre-Process Script](#pre-process-script)
+  - [Post-Process Script](#post-process-script)
+  - [Rules](#rules)
+---
+
+**This package contains a function that searches using your IPVOID account/subscription with the given query**
+
+ ![screenshot](./screenshots/screen1.png)
+
+The function makes use of the IPVOID's Rest API calls to get information on a given query, followings are the available services :-
+-	IP Reputation `/iprep/v1/pay-as-you-go/`
```

Comment:
Yes they have different pricing but the API is consistent since whatever price you choose..all you will get credit points and you burn those points by hitting different REST URLs.